src/index.js

```diff
@@ -325,17 +331,32 @@ class ReactPlayerLoader extends React.Component {
       const previous = prevProps[key];
       const current = this.props[key];
 
+      // Do not compare functions
+      if (typeof current === 'function') {
+        return acc;
+      }
+
+      if (typeof current === 'object') {
```

Comment:
Should make sure we don't error with `null` here because `typeof null === 'object'`.